File: Trabalho-Logica-de-Programação/Python/questao1.py
velocidade = int(input("Digite a velocidade: "))
velocidadeMax = int(input("Digite a velocidade maxima da avenida: "))

multa = (velocidade - velocidadeMax) * 5

if(velocidade > velocidadeMax):
    print("Sua velocidade foi acima da permitida")
    print(f'Valor da multa: {multa}R$')
else:
    print("Sua velocidade esta correta!")

File: Trabalho-Logica-de-Programação/C/questao1.c
#include <stdio.h>
#include <stdlib.h>

int main(){
    int velocidade, multavalor, velocidadeMax;

    printf("Digite a velocidade: ");
    scanf("%d", &velocidade);
    printf("Digite a velocidade maxima da avenida: ");
    scanf("%d", &velocidadeMax);

    if(velocidade > velocidadeMax){
        multavalor = (velocidade - velocidadeMax) * 5;
        printf("Velocidade acima da permitida a multa foi de %d R$", multavalor);
    }else{
        printf("Velocidade correta! Nenhuma multa sera aplicada!");
    }
    
    return 0;
}